bin/uat_deploy

```diff
@@ -14,7 +14,7 @@ deploy() {
                 --namespace=${K8S_NAMESPACE} \
                 --values ./deploy/helm/values-uat.yaml \
                 --set postgresql.image.repository=bitnamilegacy/postgresql \
-                --set postgresql.global.security.allowInsecureImages=true \
```

Comment:
Would this not allow insecure images in other namespaces?  Seems a bit insecure!

Reply:
This [section kind of implies something similar regarding globals](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values) but still not completely clear

Reply:
am going to close this as i am seeing more `postgres.global.security.allowInsecureImages` in the "world/github" and i have been unable to test this to my satisfaction. This is more cosnistent with CFE-civil approach

I note that bitnami redis in civil apply is a sufficiently low chart version that it does not care about this btinami specific parameter because it was introduced after the chart version that implemented it.